yogstation/code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -3,3 +3,41 @@
 
 /obj/item/clothing/suit/space/hardsuit/security
   jetpack = /obj/item/tank/jetpack/suit
+
+//POWERARMORS
+//Currently are no different from normal hardsuits, except maybe for the higher armor ratings.
+/obj/item/clothing/head/helmet/space/hardsuit/powerarmor
+	alternate_worn_icon = 'yogstation/icons/mob/head.dmi'
+	icon = 'yogstation/icons/obj/clothing/hats.dmi'	
+/obj/item/clothing/suit/space/hardsuit/powerarmor
+	alternate_worn_icon = 'yogstation/icons/mob/suit.dmi'
+	icon = 'yogstation/icons/obj/clothing/suits.dmi'
+
+/obj/item/clothing/head/helmet/space/hardsuit/powerarmor/t45b
+	name = "Salvaged T-45b helmet"
+	desc = "It's a pre-War power armor helmet, recovered and maintained by NCR engineers."
+	icon_state = "t45bhelmet"
+	item_state = "t45bhelmet"
+	armor = list("melee" = 50, "bullet" = 48, "laser" = 25, "energy" = 25, "bomb" = 48, "bio" = 100, "rad" = 50, "fire" = 50, "acid" = 0) 
+/obj/item/clothing/suit/space/hardsuit/powerarmor/t45b
+	name = "Salvaged T-45b power armor"
+	desc = "It's a set of T-45b power armor recovered by the NCR during the NCR-Brotherhood War."
+	icon_state = "t45bpowerarmor"
+	item_state = "t45bpowerarmor"
+	armor = list("melee" = 50, "bullet" = 48, "laser" = 25, "energy" = 25, "bomb" = 48, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
+	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/powerarmor/t45b
+
+/obj/item/clothing/head/helmet/space/hardsuit/powerarmor/advanced
+	name = "Advanced power helmet"
+	desc = "It's an advanced power armor Mk I helmet, typically used by the Enclave. It looks somewhat threatening."
+	icon_state = "advhelmet1"
+	item_state = "advhelmet1"
+	armor = list("melee" = 50, "bullet" = 48, "laser" = 25, "energy" = 25, "bomb" = 48, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
```

Comment:
0 acid resistance for a suit of power armor hmmm really makes you think also same with the other one

Reply:
yeah they should acid resist